Modelica/Magnetic/FluxTubes/Shapes/HysteresisAndMagnets/GenericHystTellinenEverett.mo

```diff
@@ -13,7 +13,7 @@ protected
                                                      mat.Hsat,-mat.Hsat, mat, false)
     "Saturation polarization";
   //final parameter Real mu0 = mat.K * mu_0;
-  parameter SI.MagneticFluxDensity eps=mat.M/1000;
+  parameter SI.MagneticFluxDensity eps=unitT*mat.M/1000;
```

Comment:
Clearly the model is unit-incorrect at the moment, and this silences the errors - but as indicated I think we are already using a lot of multiplications with unitT (and this adds another) to hide an underlying error.

Reply:
```suggestion
  parameter SI.MagneticFluxDensity eps=Js/1000;
```
From #4241 - This is clearly unit-correct and the minor change in value shouldn't matter, and we can then later clean up the weird unit of mat.M - instead of patching it here.

Reply:
I'll just trust you on this one, and leave it to the other reviewers to accept or reject.